idaes/models/properties/modular_properties/state_definitions/FTPx.py

```diff
@@ -469,6 +469,8 @@ def state_initialization(b):
                 K = None
                 break
 
+    # Default is no initialization of VLE
+    vap_frac = None
```

Comment:
No - this was from issue #1424. Later steps expect `vap_frac` to exist, so this is making sure it will exist no matter what course gets taken in the `if/else` block.

Reply:
Ah, I see. I looked at the different branches of the inner if statement, but forgot it was nested inside an outer if statement.